setup.cfg

```diff
@@ -21,21 +21,11 @@ exclude =
 # For more info: https://en.wikipedia.org/wiki/Cyclomatic_complexity#Interpretation
 max-complexity = 10
 
-[tool:pytest]
+[tool.pytest]
+minversion = 6.0
+addopts = -ra -q
 testpaths = tests
-norecursedirs =
-    migrations
 
-python_files =
```

Comment:
Love losing lines! But please let me know why we don't need these anymore

Reply:
The removed lines were generated by Cookiecutter when the skeleton of the package was initialized. We don't need all those settings and it is better to remove them to avoid any confusions.